client/src/components/Footer/Footer.js

```diff
@@ -1,12 +1,74 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Footer.css';
+import Hog from './img/hog.png';
 
 function Footer() {
   return (
     <div className="footer-component">
-      <h1>This is footer.</h1>
-    </div>
+      {/* Contact Us Section Starts Here */}
+      <div className="container">
+         <div className="row">
+           <div className="col-12 text-center text-white my-3">
+             <hr></hr>
+             <h3> Contact Us</h3>
+             <div className="mt-5 mb-3">
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://"><i  className="iconer fab fa-linkedin fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://github.com/houseofgeeks"><i  className=" iconer fab fa-github fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="mailto:iiitranchi@contact.info"><i  className="iconer fas fa-envelope fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.facebook.com/search/top?q=house%20of%20geeks%20-%20technical%20society%20of%20iiit%20ranchi"><i  className="iconer fab fa-facebook fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.instagram.com/hg.iiitranchi/"><i  className="iconer fab fa-instagram fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.youtube.com/channel/UCvbh6phGLwpOnoM8g_GwlmQ"><i  className="iconer fab fa-youtube fa-3x"></i></a>
+             </div>
+           </div>
+         </div>
+      </div>
+      {/* Contact Us Section Ends here */}
+    
+    {/* Footer starts here */}
+    <footer className="footer pb-0">
+        <div className="container">
+            <div className="row justify-content-center">
+                <div className="col-12 col-md-3 offset-md-1 links">
+                        <h5 className="font-weight-bold">Links</h5>
+                        <a className="  text-white"  href="https://">Home</a><br></br>
+                        <a className="  text-white"  href="https://">About</a><br></br>
+                        <a className="  text-white"  href="https://">Socities</a><br></br>
+                        <a className="  text-white"  href="https://">Projects</a><br></br>
+                        <a className="  text-white"  href="http://iiitranchi.ac.in/">IIITRanchi</a>
```

Comment:
Suggestion: add a `target="_blank"` and `rel="noopener noreferrer"` to the anchor tag.

Reply:
I have added <Link> tag from react-router-dom for this. And thanks for your suggestions.